pom.xml

```diff
@@ -339,7 +346,7 @@
                     <dependency>
                         <groupId>com.pinterest</groupId>
                         <artifactId>ktlint</artifactId>
-                        <version>0.49.1</version>
+                        <version>0.47.1</version>
```

Comment:
Hvorfor har vi degradert ktlint?

Reply:
Tror bare det ikke har blitt merget riktig fra master. Men takk for heads up, bumpet disse avhengighetene nå.